### PR TITLE
Prepare for release 1.0.10.4

### PR DESCRIPTION
**Pull request by yophilav**
# 1.0.10.4 (2020-12-18)
### Bug Fixes
* Fix vulnerability issues in ARM-based docker images [6603778](https://github.com/Azure/iotedge/commit/6603778d2c4dbb63aea4627f66feb5920eede3c6)
* GetModuleLogs works with http endpoints [cf176be](https://github.com/Azure/iotedge/commit/cf176be66bab28b032f0069a445ed21cc9fcdfb9)

### Bug Fixes
* Fix `edgehub_queue_length` metric [2861dbc](https://github.com/Azure/iotedge/commit/2861dbc63757787ef52897d9acc732430b46313b)
* Introduce new metrics [40b2de9](https://github.com/Azure/iotedge/commit/40b2de93787140c52c8baee36860a6eb833e03d5)
* Fix vulnerability issues in Linux ARM and Windows AMD docker images [73fa197](https://github.com/Azure/iotedge/commit/73fa1976367559d38b6392f86e625906da1117bd)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# 1.0.10.4 (2020-12-18)
+## Edge Agent
+### Bug Fixes
+* Fix vulnerability issues in ARM-based docker images [6603778](https://github.com/Azure/iotedge/commit/6603778d2c4dbb63aea4627f66feb5920eede3c6)
+* GetModuleLogs works with http endpoints [cf176be](https://github.com/Azure/iotedge/commit/cf176be66bab28b032f0069a445ed21cc9fcdfb9)
+
+## Edge Hub
+### Bug Fixes
+* Fix `edgehub_queue_length` metric [2861dbc](https://github.com/Azure/iotedge/commit/2861dbc63757787ef52897d9acc732430b46313b)
```
Comment (ancaantochi) [inline CHANGELOG.md:9]: It seems it is missing some commits: 
- darobs merge -- not sure how to summarize it
- like sha256 thumbprint support
- fix clean processor

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+* Rebase Edge on K8s [c94bc73](https://github.com/Azure/iotedge/commit/c94bc733c05d91acfd77c87dcaf6b701601985fb)
+* Fix vulnerability issues in ARM-based docker images [6603778](https://github.com/Azure/iotedge/commit/6603778d2c4dbb63aea4627f66feb5920eede3c6)
+* GetModuleLogs works with http endpoints [cf176be](https://github.com/Azure/iotedge/commit/cf176be66bab28b032f0069a445ed21cc9fcdfb9)
+
+## Edge Hub
+### Bug Fixes
+* Rebase Edge on K8s [c94bc73](https://github.com/Azure/iotedge/commit/c94bc733c05d91acfd77c87dcaf6b701601985fb)
+* Support SHA256-based for thumbprint authentication [a88c72e](https://github.com/Azure/iotedge/commit/a88c72ebbfbd2ed88c8f6c8b41060b19e129c9d9)
+* Fix `edgehub_queue_length` metric [2861dbc](https://github.com/Azure/iotedge/commit/2861dbc63757787ef52897d9acc732430b46313b)
+* Introduce new metrics [40b2de9](https://github.com/Azure/iotedge/commit/40b2de93787140c52c8baee36860a6eb833e03d5)
```
Comment (and-rewsmith) [inline CHANGELOG.md:13]: Is this a feature and not a bug fix?

**Review by dylanbronson — APPROVED** [review]
lgtm!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
+# 1.0.10.4 (2020-12-18)
+## Edge Agent
+### Bug Fixes
+* Rebase Edge on K8s [c94bc73](https://github.com/Azure/iotedge/commit/c94bc733c05d91acfd77c87dcaf6b701601985fb)
```
Comment (varunpuranik) [inline CHANGELOG.md:4]: @darobs / @veyalla - can you pls sign off on this?

Comment (varunpuranik) [reply CHANGELOG.md:4]: Same for all components below.

**Review by varunpuranik — APPROVED** [review]

**Review by dylanbronson — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (darobs) [reply CHANGELOG.md:4]: After the fact - sorry @varunpuranik, @yophilav, but there should be no functional change in edgeHub and minor changes to non-k8s mode iotedged and edgeAgent. This was intentionally merged to bring 1.0.10 up to date with k8s public preview and vice-versa.

**Pull request merged**